src/components/community-link/community-link.js

```diff
@@ -0,0 +1,52 @@
+import '@brightspace-ui/core/components/link/link.js';
+
+import { getDocumentLocaleSettings } from '@brightspace-ui/intl/lib/common.js';
+
+const Link = customElements.get('d2l-link');
+
+class CommunityLink extends Link {
+
+	static get properties() {
+		return {
+			...Link.properties,
+			articleMap: { attribute: 'article-map', type: Object }
```

Comment:
Yeah not sure how I feel about this API but can't really think of anything better. 🤔

Reply:
Yeah unfortunatly the new community doesn't really make this easy...

Reply:
alternatively it could use data-lang-<code> like
```
<d2l-labs-link-community
  data-lang-en="11064"
  data-lang-fr-CA="89026"
></d2l-labs-link-community>
```

Reply:
Yeah that feels more difficult for consumers which will likely have the map handy anyway eh. The only other thing I can think of is if we centralized all the community links somewhere behind an API. Then consumers could go `<d2l-labsl-link-community key="engagement-dashboard-help">`, the component would reach out to the API (passing the active language) and the API would return the URL. _Definitely_ more complicated.

Reply:
Yeah I don't think even community has this api. It doesn't seem to understand translated articles are "the same" given that switching the translation on an article will just redirect you to the homepage in the new language